nifi-registry-resources/src/main/resources/conf/logback.xml

```diff
@@ -68,6 +68,11 @@
     
     <logger name="org.apache.nifi.registry" level="INFO"/>
 
+    <!-- To see SQL statements set this to DEBUG -->
+    <logger name="org.hibernate.SQL" level="INFO" />
+    <!-- To see the values in SQL statements set this to TRACE -->
+    <logger name="org.hibernate.type" level="INFO" />
+
```

Comment:
When I ran the app. I noticed Hibernate log output going to bootstrap.log instead of app.log. Is that intended?

Example output in bootstrap.log:

```
2017-09-20 13:27:34,585 INFO [NiFi logging handler] org.apache.nifi.registry.StdOut Hibernate: select bucketenti0_.id as id1_0_0_, bucketenti0_.created as created2_0_0_, bucketenti0_.description as descript3_0_0_, bucketenti0_.name as name4_0_0_ from BUCKET bucketenti0_ where bucketenti0_.id=?
```

Reply:
Good find, will investigate